statsmodels/distributions/discrete.py

```diff
@@ -161,6 +163,65 @@ def convert_params(self, mu, alpha, p):
     longname='Zero Inflated Generalized Negative Binomial')
 
 
+class truncatedpoisson_gen(rv_discrete):
+    '''Truncated Poisson discrete random variable
+    '''
+    # TODO: need cdf, and rvs
+
+    def _argcheck(self, mu, truncation):
+        # this does not work
+        # vector bound breaks some generic methods
+        # self.a = truncation + 1 # max(truncation + 1, 0)
+        return (mu >= 0) & (truncation >= 0)
```

Comment:
we can allow here `(truncation >= -1)` so we include the no truncation case
that seems to work.